### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by lbrenne**
Update limit for Assignment and Classnotebook from 200 to 300

### CODE REVIEW

**Comment by lbrenne** [comment]
Updated new limits

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -211,8 +211,8 @@ Limits for class teams are listed in the following table:
 |Feature  |Maximum limit  |
 |---------|---------|
 |Number of members in a team    | See the [Teams and channels](#teams-and-channels) section of this article        |
-|Number of members to use Assignments in a class team    | 200        |
-|Number of members to use a OneNote Class Notebook in a class team     |200         |
+|Number of members to use Assignments in a class team    | 300        |
+|Number of members to use a OneNote Class Notebook in a class team     |300         |
 
 A class team can support more than 200 members. However, if you plan to use either the Assignments app or Class Notebook app within your team, you will need to keep the number of members below the maximum limits above.
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:217]:
```suggestion
A class team can support more than 300 members. However, if you plan to use either the Assignments app or Class Notebook app within your team, you will need to keep the number of members below the maximum limits above.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lbrenne, please also commit the suggested change in line 217. Many thanks.

**Comment by lbrenne** [comment]
Updated

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by lbrenne** [comment]
Can we get this PR published out on Docs

Leif

**Comment by lbrenne** [comment]
Sorry if this information somewhere, but when does the doc go from live to actually Live on the docs site

**Comment by JohanFreelancer9** [comment]
@lbrenne - Should be live on the docs site today.

**Pull request merged**